packages/map-template/CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.60.0] - 2024-12-05
+
+- Added `MAX_RESULTS` constant to limit the maximum number of search results displayed at once 
```

Comment:
Maybe state what the max result is instead of mentioning the constant. Something like:

`- Limit the maximum number of search results to 100 to improve render performance.`